hail_scripts/reference_data/combine.py

```diff
@@ -46,6 +47,18 @@ def get_select_fields(selects, base_ht):
                 select_fields[key] = ht
     return select_fields
 
+def get_enum_select_fields(enum_selects, base_ht):
```

Comment:
I have (sorta) lost faith that trying to configure the enum selects with a function like this (vs. using a custom select) is worthwhile.  

The more enum selects that get added the more options this function needs to support which is no bueno.  

One potential solution is to do two passes at the select... one which would `custom_select` and handle any field specific parsing and then a second select to do a straightforward enum mapping.

Reply:
Clinvar can _maybe_ fit into this... it might be _too_ custom though.

Reply:
I think this has gotten complex enough that its not super readable, and I think the two pass approach you propose might be the better solution